app/src/main/java/com/tari/android/wallet/ui/fragment/onboarding/CreateEmojiIdFragment.kt

```diff
@@ -0,0 +1,347 @@
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE
+ * OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package com.tari.android.wallet.ui.fragment.onboarding
+
+import android.animation.*
+import android.content.Intent
+import android.os.Bundle
+import android.os.Handler
+import android.view.View
+import android.widget.*
+import androidx.core.view.marginTop
+import butterknife.BindDimen
+import butterknife.BindView
+import butterknife.OnClick
+import com.airbnb.lottie.LottieAnimationView
+import com.daasuu.ei.Ease
+import com.daasuu.ei.EasingInterpolator
+import com.tari.android.wallet.R
+import com.tari.android.wallet.ui.activity.AuthActivity
+import com.tari.android.wallet.ui.fragment.BaseFragment
+import com.tari.android.wallet.util.Constants
+
+class CreateEmojiIdFragment : BaseFragment() {
+
+    @BindView(R.id.tari_wallet_anim)
+    lateinit var tariAnimationView: LottieAnimationView
+    @BindView(R.id.rootView)
+    lateinit var rootView: FrameLayout
+    @BindView(R.id.txt_hello)
+    lateinit var helloText: TextView
+    @BindView(R.id.txt_just_sec_desc)
+    lateinit var justSecDescText: TextView
+    @BindView(R.id.txt_just_sec_title)
+    lateinit var justSecTitle: TextView
+    @BindView(R.id.hello_text_back_view)
+    lateinit var helloTextBackView: View
+    @BindView(R.id.checkmark_anim)
+    lateinit var checkMarkAnim: LottieAnimationView
+    @BindView(R.id.txt_wallet_address_des)
+    lateinit var walletAddressDescText: TextView
+    @BindView(R.id.creat_emoji_btn)
+    lateinit var createEmojiIdButton: Button
+    @BindView(R.id.just_sec_back_view)
+    lateinit var justSecBackView: View
+    @BindView(R.id.nerd_face_emoji)
+    lateinit var nerdFaceEmoji: LottieAnimationView
+    @BindView(R.id.txt_create_your_emoji_id)
+    lateinit var createYourEmojiIdText: TextView
+    @BindView(R.id.create_emoji_id_text_back_view)
+    lateinit var createEmojiIdTextBackView: View
+    @BindView(R.id.txt_awesome)
+    lateinit var awesomeText: TextView
+    @BindView(R.id.white_bg_view)
+    lateinit var whiteBgView: View
+    @BindView(R.id.awesome_text_back_view)
+    lateinit var awesomeTextBackView: View
+    @BindView(R.id.main_img_small_gem)
+    lateinit var smallGem: ImageView
+
+    @BindDimen(R.dimen.create_emoji_id_button_bottom_margin)
+    @JvmField
+    var createEmojiButtonBottomMargin = 0
+
+    val uiHandler = Handler()
+
+    override val contentViewId = R.layout.fragment_create_emoji_id
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        init()
+    }
+
+    override fun onDestroy() {
+        super.onDestroy()
+        uiHandler.removeCallbacksAndMessages(null)
+    }
+
+    private fun init() {
+        whiteBgView.translationY = -whiteBgView.height.toFloat()
+
+        rootView.post { startTariViewAnimation() }
```

Comment:
There's a jump in size of the TARI logo while switching to this fragment from the previous - it's not continuous.